metadata.rb

```diff
@@ -14,7 +14,7 @@
 supports 'scientific'
 supports 'oracle'
 
-depends  'python'
+depends  'poise-python'
```

Comment:
@t2y Here is where you can include `, '~> 1.5.1'` (like the runit line below it). Do this here and you'll want to remove the line from Berkshire.